projects/storefrontapp-e2e-cypress/cypress/integration/regression/customer-ticketing/view-ticket-details.e2e-spec.ts

```diff
@@ -22,6 +27,101 @@ describe('ticket details', () => {
         customerTicketing.clickTicketInRow();
         customerTicketing.verifyTicketDetailsPageVisit();
       });
+
+      it('clicking a ticket should open its corresponding ticket details', () => {
+        customerTicketing.loginRegisteredUser();
+        customerTicketing.clickMyAccountMenuOption();
+        customerTicketing.clickCustomerSupportMenuOption();
+        customerTicketing.verifyTicketListingPageVisit();
+        customerTicketing.createTicket({
+          subject: 'First ticket',
+          message: 'First ticket',
+          category: TestCategory.complaint,
+        });
+        customerTicketing.visitElectronicTicketListingPage();
+        customerTicketing.verifyTicketListingPageVisit();
+        customerTicketing.createTicket({
+          subject: 'Second ticket',
+          message: 'Second ticket',
+          category: TestCategory.complaint,
+        });
+        customerTicketing.visitElectronicTicketListingPage();
+        customerTicketing.verifyTicketListingPageVisit();
+        let ticketDetails = customerTicketing.extractTicketDetailsFromFirstRowInTicketListingPage();
+        customerTicketing.clickTicketInRow();
```

Comment:
Please pass a Ticket Position constant to have clarity on which item was clicked